docs/README.asciidoc

```diff
@@ -2,3 +2,37 @@ The Elasticsearch docs are in AsciiDoc format and can be built using the Elastic
 
 See: https://github.com/elastic/docs
 
+Snippets marked with `// AUTOSENSE` are automatically annotated with "VIEW IN
+SENSE" in the documentation and are automatically tested by the command
+`gradle :docs:check`. You can manipulate the test exeuction in the following
+ways:
+
+* `// TEST`: Explicitly marks a snippet as a test. Snippets marked this way
+are tests even if they don't have `// AUTOSENSE`.
+  * `// TEST[s/foo/bar/]`: Replace `foo` with `bar` in the test. This should be
+  used sparingly because it makes the test "lie". Sometimes, though, you can use
+  it to make the tests more clear.
+  * `// TEST[catch:foo]`: Used to expect errors in the requests. Replace `foo`
+  with `request` to expect a 400 error, for example.
+  * `// TEST[continue]`: Continue the test started in the last snippet. Between
```

Comment:
maybe call this `continuation` instead of `continue`, because when I first read it I thought of continue within a loop.